plugins/microsoft_log_analytics/plugin.spec.yaml

```diff
@@ -141,6 +141,48 @@ actions:
         type: string
         required: true
         example: "AzureActivity | summarize count() by Category"
+    output:
+      tables:
+        title: Tables
+        description: Array of tables representing the query result. Each table contains name, columns, and rows keys
```

Comment:
https://docs.rapid7.com/insightconnect/style-guide#punctuation
Try changing the description to:
`Array of tables representing the query result, with each table containing a name, column and row keys`